integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/command/SICreationCommand.scala

```diff
@@ -390,8 +391,13 @@ private[sql] case class CarbonCreateSecondaryIndexCommand(
       }
       val indexTablePath = CarbonTablePath
         .getMetadataPath(tableInfo.getOrCreateAbsoluteTableIdentifier.getTablePath)
+      val listOfLoadFolderDetailsArrayMainTbl: Array[LoadMetadataDetails] =
```

Comment:
`listOfLoadFolderDetailsArrayMainTbl` variable name is big and not clear, can give a meaningful name like `mainTblLoadMetadataDetails`